neo/Consensus/ConsensusContext.cs

```diff
@@ -43,6 +43,8 @@ internal class ConsensusContext : IDisposable, ISerializable
         private readonly Store store;
         private readonly Random random = new Random();
 
+        private bool nextBlockLight = false;
```

Comment:
What do you mean by "nextBlockLight"?

Reply:
I want to produce a block with half of transactions in the next block, 250 if the block max is 500. why? because is easier for the other CN to receive 250 TX than 500, and the view change was because this.

Reply:
@shargon can we rename it to "reduceNextBlockSize" or something 'equivalent'?

Reply:
Yes, but first decide if it's useful, it is for me hahha